utils/build-script-impl

```diff
@@ -1491,19 +1516,26 @@ function set_swiftpm_bootstrap_command() {
 
 
 for host in "${ALL_HOSTS[@]}"; do
+    # Skip this pass when the only action to execute can't match.
+    if [[ "${ONLY_EXECUTE}" != "" ]] && ! [[ "${ONLY_EXECUTE}" == ${host}-*-build ]]; then
```

Comment:
You can also use `!=` instead of `! [[ ... == ... ]]`, I think it is more readable that way.